slack-base/src/main/java/com/hubspot/slack/client/models/ConversationsRepliesMessageIF.java

```diff
@@ -0,0 +1,55 @@
+package com.hubspot.slack.client.models;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.Optional;
+
+import org.immutables.value.Value.Default;
+import org.immutables.value.Value.Immutable;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.PropertyNamingStrategy.SnakeCaseStrategy;
+import com.fasterxml.jackson.databind.annotation.JsonNaming;
+import com.hubspot.immutables.style.HubSpotStyle;
+import com.hubspot.slack.client.models.files.SlackFile;
+
+@Immutable
+@HubSpotStyle
+@JsonNaming(SnakeCaseStrategy.class)
+public interface ConversationsRepliesMessageIF {
```

Comment:
Reason for this is that Slack has said the `replies` deprecation is only applicable to the `conversations.replies` endpoint, so we're narrowly making the change here. `LiteMessage` is used in several places, so I didn't want to make a change that could adversely affect the other places.